client-side/src/download as excl/downloadAsExcel.jsx

```diff
@@ -0,0 +1,16 @@
+
+
+import { saveAs } from 'file-saver';
+import * as XLSX from 'xlsx';
```

Comment:
I don't see any change in package.json, you are using here a new libraries they should be added to package.json